app/src/main/res/values/strings.xml

```diff
@@ -279,6 +279,8 @@
     <!-- This is the title of the edit login page. -->
     <string name="edit_login_title">Edit Login</string>
 
+    <string name="undisplayed_credential_mutation_error" translatable="false">Undisplayed</string>
```

Comment:
Prefer "hidden" to "undisplayed". Willing to be convinced otherwise 😉

Reply:
:+1: